students/jeff_shabani/session09/mailroom_oo/oo_mailroom_tests.py

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env python3
+
+# Jeff Shabani
+# March 2019
+# Python 210, Session 9
+# donor_models.py
+
+import mock
+import unittest
+
+from io import StringIO
+from cli_main import *
+from pathlib import Path
+from unittest.mock import patch
+
+ANSWER = 'New_Donor'
+AMOUNT = 4512
+
+donors_test = {'Karsten Willems': [120, 130, 50],
+               'Sammy Maudlin': [500, 125, 670, 1000]
+               }
+# use for testing setting new letter path
+base_path = Path(r'C:\JRS\Python\UW\Intro_Klass\students\jeff_shabani\session09\mailroom_oo')
```

Comment:
This will cause your tests to fail anywhere but in your computer. Can you change it so that it works on any system?